iphone/CoreApi/CoreApi/Bookmarks/MWMBookmarksManager.mm

```diff
@@ -408,8 +408,9 @@ - (MWMBookmarkGroupType)getCategoryGroupType:(MWMMarkGroupID)groupId {
       return MWMBookmarkGroupTypeCollection;
     case kml::CompilationType::Day:
       return MWMBookmarkGroupTypeDay;
+    default:
+      return MWMBookmarkGroupTypeRoot;
   }
-  return MWMBookmarkGroupTypeRoot;
```

Comment:
It's not a good way to suppress this warning. If I will add new option like MWMBookmarkGroupTypeXXX, I will **not** get any warning here. If compiler gives warning here and we process **all** case variants here, we should do something else ..